kolibri/content/constants/presets.py

```diff
@@ -0,0 +1,29 @@
+"""
+This module contains constants representing the ids of FormatPreset.
+"""
+from django.utils.translation import ugettext_lazy as _
+
+# constants for FormatPreset
+HIGH_RES_VIDEO = "high_res_video"
+MID_RES_VIDEO = "mid_res_video"
+LOW_RES_VIDEO = "low_res_video"
+VECTOR_VIDEO = "vertor_video"
+
+EN_SUBTITLE = "en_subtitle"
+ES_SUBTITLE = "es_subtitle"
+ZH_SUBTITLE = "zh_subtitle"
```

Comment:
> when a user create a video content and click the subtitle dropdown list, how the template know how many subtitles to show if we don't use preset for subtitles?

If the preset has `multi_language` set, then it loops over all the languages in the `Language` table to populate the dropdown.

> distinguish between the transcript of the video dialog in its original language, and subsequent translations of that transcript

In Kolibri, videos in different languages would typically live in different channels (or at least, under different content nodes), so the literal transcript would be the one in the same language as the video.

> jump ahead the a11y-curve and adopt the term captions

That could be good. VTT also supports things like position on screen, formatting, etc.

"Captions on the other hand, may be placed in different locations on the screen in order to make clear to the audience who is speaking. This is especially useful for deaf individuals who can’t rely on voice distinctions to pinpoint the speaker."
https://www.alsintl.com/blog/subtitles-captions-difference/

@66eli77 -- let's call this preset "captions"